node/CHANGELOG.md

```diff
@@ -15,7 +15,7 @@ All notable changes to this project will be documented in this file.  The format
 ### Added
 
 * Add `net-info` command to diagnostics port to gain insights into the nodes network connectivity state.
-* Add metrics `deploy_buffer_total_deploys`, `deploy_buffer_held_deploys`, `deploy_buffer_dead_deploys` to report status of the node deploy buffer, `block_accumulator_block_acceptors`,  `block_accumulator_known_child_blocks` to track status of the block accumulator component, `{forward|historical}_block_sync_time` histogram metric to track the progress of block synchronization, `chain_low_seq_block_height` to report the lowest height that has been synchronized continuously from the tip.
+* Add metrics `deploy_buffer_total_deploys`, `deploy_buffer_held_deploys`, `deploy_buffer_dead_deploys` to report status of the node deploy buffer, `block_accumulator_block_acceptors`, `block_accumulator_known_child_blocks` to track status of the block accumulator component, `{forward|historical}_block_sync_time` histogram metric to track the progress of block synchronization, `chain_low_seq_block_height` to report the lowest height that has been synchronized continuously from the tip, `sync_leap_duration`, `sync_leap_fetched_from_peer`, `sync_leap_rejected_by_peer`, `sync_leap_cant_fetch` to track progress of the sync leaper component.
```

Comment:
It could be interesting to see the total number of sync leaps, but actually, such metric would belong to reactor, not here.
Anyway, no big deal :)

Reply:
We can actually see the total number of successful sync leaps thanks to `the sync_leap_duration` Histogram which generates a `sync_leap_duration_count` metric. What we don't have metrics though for is the number of unsuccessful sync leaps. Maybe it's worth adding that.

Reply:
True, I keep forgetting that histograms come with count :)
I think successful ones are enough.